plugins/flashrom/flashrom.quirk

```diff
@@ -6,3 +6,8 @@ DeviceId=librem15v3
 [HwId=52b68c34-6b31-5ecc-8a5c-de37e666ccd5]
 DeviceId=LT1000
 VersionFormat=quad
+
+# Star LabTop Mk IV
+[HwId=baf1d04e-fd16-5e6a-93cc-1c23d171f879]
+DeviceID=LabTop
+VersionFormat=triplet
```

Comment:
is this really a triplet?  I'm just confirming since the other systems in this file don't use a triplet.